asmcomp/cmmgen.ml

```diff
@@ -3082,6 +3082,31 @@ let global_data name v =
   Cdata(emit_structured_constant (name, Global)
           (Uconst_string (Marshal.to_string v [])) [])
 
+let module_map_debug = try
+                         ignore (Sys.getenv "CAML_MODMAP_DEBUG"); true
+  with Not_found -> false
```

Comment:
Could we maybe remove that part? I have nothing against having debugging code below, but I dislike the proliferation of undocumented environment variables in the compiler.